.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+# Description
```

Comment:
Do we want to add a reference to linear ?

Reply:
let's look at this as we go forward...